pkg/auth/zcapld/service.go

```diff
@@ -108,7 +108,7 @@ func (s *Service) Handler(resourceID string, w http.ResponseWriter, next http.Ha
 		&zcapld.InvocationExpectations{
 			Target:         resourceID,
 			RootCapability: rootCapability.ID,
-			Action:         "read",
+			Action:         "write",
```

Comment:
The expected action depends on the invocation... two may be acting on the same resource, but if one is a `GET` and the other is a `POST`, they can't have the same action (if we're following RESTful semantics).

If an application's handler is registered for an HTTP GET, then generally its action would be "read". If it's registered for a POST, then generally its action would be "write".

Reply:
done